test/suites/storage_buckets.sh

```diff
@@ -101,6 +101,14 @@ test_storage_buckets() {
   roAccessKey=$(echo "${creds}" | awk '{ if ($1 == "Access" && $2 == "key:") {print $3}}')
   roSecretKey=$(echo "${creds}" | awk '{ if ($1 == "Secret" && $2 == "key:") {print $3}}')
 
+  # Test creating a bucket key with YAML bucket key config.
+  creds=$(lxc storage bucket key create "${poolName}" "${bucketPrefix}.foo" yaml-key < <(cat <<EOF
+role: read-only
+EOF
+))
```

Comment:
I think the subshell and cat are not strictly required.
```suggestion
  creds=$(lxc storage bucket key create "${poolName}" "${bucketPrefix}.foo" yaml-key << EOF
role: read-only
EOF
)
```

Reply:
fixed